### PR TITLE
implement percentile for jax

### LINKED ISSUE #19082: percentile

### PR DESCRIPTION
**Pull request by Mirzonabot**
Close #19082

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by hello-fri-end** [comment]
> > If you are working on an open task, please **edit** the PR description to link to the issue you've created.
> > For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).
> > Thank you hugs
> 
> Just did it. Can you review the PR now please?

Hey @Mirzonabot ! Thanks for opening this PR. Could you please add a test for the same? You can take a look at https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#ivy-frontend-tests to get an idea of how to add a test. Feel free to request another review when you are done with the changes :pray:

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
You should not push any changes to `.idea/vcs.xml` file. Can you please revert these changes?

File: ivy/functional/frontends/jax/numpy/statistical.py [hunk 1]
```diff
@@ -486,3 +486,9 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=N
     return ivy.cov(
         m, y, rowVar=rowvar, bias=bias, ddof=ddof, fweights=fweights, aweights=aweights
     )
+
+# percentile
+
+@to_ivy_arrays_and_back
+def percentile(a, q, axis=None, out=None, overwrite_input=False, interpolation="linear", keepdims=False):
```
Comment (hello-fri-end) [inline ivy/functional/frontends/jax/numpy/statistical.py:493]: The function signature doesn't match with jax's documentation of percentile (https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.percentile.html)

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Mirzonabot ! Just left a minor comment, can you please resolve it? Once you have updated the function signature, could you also update the test to generate values for all the parameters in the function signature - we should always make sure that our tests are exhaustive and cover all possible cases for the input. Feel free to request another review when you're done. Thanks.

**Comment by hello-fri-end** [comment]
Hey @Mirzonabot! There isn't an implementation of `percentile` in the functional API currently. Generally, the policy is to either implement such a function as a composition in the frontends ( given that the compositional implementation isn't more than 3-4 lines) or create another issue for adding it to the functional API first. Since `percentile` implementation will definitely be more than 3-4 lines, I would advise you to create an issue for adding it to the functional API. Here are the steps you need to follow:

1. Go to the [GitHub Ivy](https://github.com/unifyai/ivy) page, select the Issues tab, and click on the green button New issue at the centre-right of the screen.
2. You will see 5 options. Each option has a predetermined form. To start filling in the form, click on the green button at the right which says Get started.
3. Choose the "Missing Function Suggestion" option. In this form, you'll be asked to fill in the reason you think Ivy should implement the suggested function, as well as the links to any native implementations of the suggested function.
4. To submit your issue, you will have to complete the requirements in the form and click on the green button Submit new issue at the right-bottom of the screen.

After you submit the issue, a member of the Ivy team will review your suggestion and they will add it as a subtask to the Ivy Experimental API Open Task. You can then either take up the task yourself or wait for someone else to implement it. 

Please let me know you if have any doubts about the same. Thanks :)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
